rules.neon

```diff
@@ -1,34 +1,208 @@
-# deprecated, you probably want to use strict-rules.neon, see README
+parameters:
```

Comment:
Reuse old rules.neon that was previously deprecated? In that case, new major version is needed.

Reply:
Not an issue in my opinion, I'd go for the 2.0 release.